src/inmanta/server/server.py

```diff
@@ -1229,10 +1229,10 @@ def resource_action_update(self, env, resource_ids, action_id, action, started,
 
         if len(messages) > 0:
             resource_action.add_logs(messages)
-            for msg in messages:
-                loglevel = const.LogLevel[msg["level"]].value
-                log_line = data.LogLine.log(loglevel, msg["msg"], *msg["args"])
-                self._write_to_resource_action_log(log_line)
+            if self._resource_action_logger:
+                for msg in messages:
+                    # all other data is stored in the database
```

Comment:
perhaps also add a comment about the interpolation problem that is subtly avoided here